governance/xc_admin/packages/xc_admin_common/src/programs/types.ts

```diff
@@ -116,3 +119,64 @@ export interface ValidationResult {
   error?: string;
   changes?: any;
 }
+
+/**
+ * Lazer feed metadata type
+ */
+export type LazerFeedMetadata = {
+  priceFeedId: number;
+  name: string;
+  symbol: string;
+  description: string;
+  assetType: string;
+  exponent: number;
+  minPublishers: number;
+  minRate: string;
+  expiryTime: string;
+  isActivated: boolean;
+  hermesId?: string;
+  cmcId?: number;
+  fundingRateInterval?: string;
+  quoteCurrency?: string;
+  marketSchedule: string;
+};
+
+/**
+ * Lazer feed type
+ */
+export type LazerFeed = {
+  metadata: LazerFeedMetadata;
+  pendingActivation?: string;
+};
+
+/**
+ * Lazer publisher type
+ */
+export type LazerPublisher = {
+  publisherId: number;
+  name: string;
+  publicKeys: string[];
+  isActive: boolean;
+};
+
+/**
+ * Full Lazer state type
+ */
+export type LazerState = {
+  shardId: number;
+  lastSequenceNo: string;
+  lastTimestamp: string;
+  shardName: string;
+  minRate: string;
+  feeds: LazerFeed[];
+  publishers: LazerPublisher[];
+};
+
+/**
+ * Lazer-specific configuration type
+ */
+export type LazerConfig = {
+  programType: ProgramType.PYTH_LAZER;
+  // The Lazer state data
+  state: LazerState;
```

Comment:
This type looks redundant, if the type is explicitly a lazer type, why would you also need a `programType: ProgramType.PYTH_LAZER` ?  This is again making me question the `program_registry.ts` pattern.